app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+import os
+
+
+def move_clear_file(file_path1: str, file_path2: str) -> None:
+    with (
+        open(file_path1, "r") as file_in,
+        open(file_path2, "w") as file_out
+    ):
+        file_out.write(file_in.read())
+        file_in.close()
```

Comment:
No need to close file manually, the close will be done after with block is ended

Reply:
To delete a file inside the context manager you have to close it manually, otherwise it won't let you delete it

Reply:
Okay, let it be

Reply:
@YehorKorn maybe then delete the file after context manager?

Reply:
@YehorKorn check this